src/index.html

```diff
@@ -7,8 +7,41 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo__header">
```

Comment:
We use 2 underlines for elements inside some blocks.

For example:

`.header`- block
`.header__logo` - element inside this block
`.header__logo--green` - modifier for this element which changes the default behavior

Reply:
Is it correct that I have to change the class name from "moyo__header" to "header__moyo" instead?